### ISSUE TITLE
How to trigger Andy fk drop bug

### ISSUE THREAD

**Issue by DarkHorseDre** [kolton/d2bot-with-kolbot#997]
My bot just quested nm andy and quit without going to act 2 and now all drops are crapz QQ

Do I need to have andy script check for quest and if quest incomplete, have it speak to warriv and travel after fk?
I'm hoping there's an quicker way!

**Comment by 5noop** [comment]
> Do I need to have andy script check for quest and if quest incomplete, have it speak to warriv and travel after fk?
> I'm hoping there's an quicker way!

Well I can't really see any other way apart from your proposition, do you have another idea ? It shouldn't be too hard to write anyway.

Now I can understand the question. but since it's something you have to do just once per char do you really think a script is needed ? I believe the usual Autoplay or Autosmurf scripts will bring your character directly to a2 after Andy.

**Comment by DarkHorseDre** [comment]
i dont have another idea, hence the post! ;) But I thought there may be some script that does it - and as you point out autoplay and autosmurf which I've never used.

Script is a good idea as I am raising 3 toons together in HC and they killed andy but not strong enough to rush through act 2, so needed them to bug andy.  as you know, bugged nm andy is sweet.
Having every andy kill ensure the bug occurs is always a good idea - I cant see why it wouldnt be that way tbh

**Comment by 5noop** [comment]
Alright why not after all. I don't have a char without andy q to test it but feel free to try this script https://pastebin.com/hkVQBwLm

**Comment by DarkHorseDre** [comment]
Thanks bro! You're a bit late with the Christmas sprit, but I'll take it! :)

I like what you're doing there.. is clearafter custom?
I like the clearlevel 37 - too much wasted when just killing andy and leaving..

**Comment by 5noop** [comment]
Yeah actually the custom part will check 1) if the char needs at least one rejuv 2) if ClearAfter is set to true, and then it will kill all elites in lvl 4 (assuming you set `Config.ClearType = 0xF;`). It's the best way I found to get rejuvs in classic since I only run andy/duriel/meph.

**Comment by DarkHorseDre** [comment]
ah, I just run random parts of ac1 to get em.

so you run classic!  how did you do ensure andy bug, manually?

**Comment by 5noop** [comment]
It depends, either manually, with Autosmurf or during a rush.
